tests/jvmoverloads/build.gradle.kts

```diff
@@ -1,12 +1,12 @@
 plugins {
-  id("org.jetbrains.kotlin.jvm")
+  kotlin("jvm")
```

Comment:
:old_man_yells_at_gradle: I personally think this is too much. I like the consistency of `id("")` over the (maybe?) type safety of `java` or `kotlin("jvm")`. The `kotlin("jvm")` especially is weird because it behaves differently in the plugins or dependencies block 🤷 

I am definitely not dying on this one hill though :).

Reply:
Ahaha no worries, consistency is always a good thing! I'll put back the `id`!

Note: you can also use the version catalog for plugins with `alias(libs.plugins.xyz)` but that only works if the version is specified, which is not what we want.